tmail-backend/apps/memory/pom.xml

```diff
@@ -316,6 +316,10 @@
                         </ports>
                         <appRoot>/root</appRoot>
                         <creationTime>USE_CURRENT_TIMESTAMP</creationTime>
+                        <jvmFlags>
+                            <!-- Ajout du répertoire extensions-jars au classpath -->
+                            <jvmFlag>-Djava.class.path=/root/extensions-jars/*</jvmFlag>
```

Comment:
I may be naive but I had `-cp /root/extensions-jars/*:/root/libs/*` in mind.

If not working another naive alternative is to drop the extension jars into `/root/libs` and no longer in `/root/extensions-jars`. 

CF recommendations for a similar problem (JDBC drivers) https://github.com/apache/james-project/blob/c80ea107886e8798f44c7e348e10f66371206b58/server/apps/jpa-app/docker-compose.yml#L18

Can you please try both of these approaches?